dds/DCPS/XTypes/DynamicDataXcdrReadImpl.cpp

```diff
@@ -590,11 +688,43 @@ bool DynamicDataXcdrReadImpl::read_value(ValueType& value, TypeKind tk)
   return false;
 }
 
+// Check if a struct member with the given Id is excluded from a sample (in case of
+// KeyOnly or NestedKeyOnly serialization).
+bool DynamicDataXcdrReadImpl::exclude_struct_member(MemberId id, DDS::MemberDescriptor_var& md) const
+{
+  DDS::DynamicTypeMember_var dtm;
+  if (type_->get_member(dtm, id) != DDS::RETCODE_OK) {
+    if (DCPS::log_level >= DCPS::LogLevel::Notice) {
+      ACE_ERROR((LM_ERROR, "(%P|%t) DynamicDataXcdrReadImpl::exclude_struct_member:"
```

Comment:
```suggestion
      ACE_ERROR((LM_NOTICE, "(%P|%t) NOTICE: DynamicDataXcdrReadImpl::exclude_struct_member:"
```

Do the same for every other notice added to this file.